pkg/operator/edgefs/cluster/target/pod_test.go

```diff
@@ -168,5 +172,6 @@ func TestHostNetwork(t *testing.T) {
 		storageSpec, "", *resource.NewQuantity(100000.0, resource.BinarySI),
 		rookalpha.Placement{}, edgefsv1alpha1.NetworkSpec{ServerIfName: "eth0"}, v1.ResourceRequirements{}, metav1.OwnerReference{})
 
+	// needResurrect as parameters
```

Comment:
It would "look" better to use inline arg comments, e.g., `c.createSetupConfigs(false /* resurrect bool */)`

Reply:
Makes total sense! Done.